SpatialGDK/Source/Private/EngineClasses/SpatialActorChannel.cpp

```diff
@@ -746,21 +746,26 @@ void USpatialActorChannel::UpdateSpatialRotation()
 
 FVector USpatialActorChannel::GetActorSpatialPosition(AActor* InActor)
 {
+	FVector Location;
+
 	// If the Actor has an Owner, use its position.
 	// Otherwise if the Actor has a well defined location then use that
 	// Otherwise use the origin
 	if (InActor->GetOwner())
 	{
-		return GetActorSpatialPosition(InActor->GetOwner());
+		Location = GetActorSpatialPosition(InActor->GetOwner());
```

Comment:
For actors with an owner, since this function gets called recursively, you will end up "rebasing" multiple times.

Reply:
Maybe ok?